base/audit_events.go

```diff
@@ -357,53 +357,78 @@ var AuditEvents = events{
 	},
 	AuditIDCreateDatabase: {
 		Name:        "Create database",
-		Description: "A new database was created",
+		Description: "A new database was created from REST API",
```

Comment:
I am OK dropping these, but I think there's some potential for confusion around creation or deletion in the sense of appearing or disappearing via the bootstrap API.

I think nearly all cases of removing a database via bootstrap are going to happen because another node called DELETE, but we have seen cases where a database is removed because of a bad cluster connection. I doubt this will be confusing.

I was confused here because does "a new database was created" mean we call `NewDatabaseContext` and the database now exists on this node, or does it mean that a REST call occurred?

I think this is a point of confusion that is not likely to occur to non Sync Gateway developers.